bin/ember

```diff
@@ -7,6 +7,11 @@ process.title = 'ember';
 var resolve = require('resolve');
 var exit = require('exit');
 
+var updateNotifier = require('update-notifier');
```

Comment:
> This should likely be somwhere in lib/cli

moving it to `lib/cli` has two issues:

- it would check the version of the project-specific `ember-cli` dependency instead of the globally installed `ember` command
- if you fix that and pass the version/package info of the global command to `lib/cli` you would only run the version check for the global command if the project-specific `ember-cli` was `^2.5.0` which doesn't make much sense IMHO

that is essentially why I put it there instead of in `lib/cli`, otherwise I fully agree that changes should usually happen there instead (e.g auto completion)